charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py

```diff
@@ -32,25 +32,11 @@ def new_representer(dumper, data):
         'min_kubernetes': '1.14.0-0'
     },
     {
-        'source': 'https://raw.githubusercontent.com/etcd-io/website/master/content/docs/current/op-guide/grafana.json',
+        'source': 'https://raw.githubusercontent.com/etcd-io/website/master/content/en/docs/v3.4/op-guide/grafana.json',
         'destination': '../templates/grafana/dashboards-1.14',
         'type': 'json',
         'min_kubernetes': '1.14.0-0'
```

Comment:
I think we could remove the whole `min_kubernetes` and `max_kubernetes` logic all together then.

Reply:
I would keep it, we might need it at some other point if things become incompatible again.